apidocs.yaml

```diff
@@ -0,0 +1,826 @@
+      operationId: submit_application_v1
+      tags:
+        - "actions"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to submit
+          required: true
+          schema:
+            type: string
+      responses:
+        "204":
+          description: Successful Response (no content)
+        "422":
+          description: Validation warnings
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ValidationWarnings"
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/create-application:
+    post:
+      summary: Create application
+      operationId: create_application_v1
+      tags:
+        - "actions"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateApplication"
+      responses:
+        "204":
+          description: Successful Response (no content)
+        "422":
+          description: Validation warnings
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ValidationWarnings"
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/cancel:
+    post:
+      summary: Cancel application
+      operationId: cancel_application_v1
+      tags:
+        - "actions"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to submit
+          required: true
+          schema:
+            type: string
+      responses:
+        "204":
+          description: Successful Response (no content)
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/copy-as-new:
+    post:
+      summary: Copy application as new
+      operationId: copy_application_as_new_v1
+      tags:
+        - "actions"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to resubmit
+          required: true
+          schema:
+            type: string
+      responses:
+        "204":
+          description: Successful Response (no content)
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/accept-terms:
+    post:
+      summary: Accept terms of use
+      operationId: accept_application_terms_v1
+      tags:
+        - "terms"
+      parameters:
+        - name: id
+          in: path
+          description: ID of application to accept terms of use for
+          required: true
+          schema:
+            type: string
+      responses:
+        "204":
+          description: Successful Response (no content)
+      security:
+        - auth:
+          - write:applications
+  /api/v1/applications/{id}/add-attachment:
```

Comment:
This one too: POST `/api/v1/applications/{id}/attachments` will be more "restful" no?

Reply:
I will close this conversation to keep it in a single thread.